ferry.go

```diff
@@ -828,6 +828,11 @@ func (f *Ferry) Progress() *Progress {
 		return true
 	})
 
+	// Verifier information
+	if f.Verifier != nil {
```

Comment:
At some point I think we should consider removing these conditionals in favor of a single runtime check that prevents operation without a verifier. 

What do you think?

Reply:
I still think we should allow operations without a verifier. It's a bit unsafe if done carelessly thus we don't really recommend it officially. However, I can see cases where we would want to run with the verifier off (if people have performed validation runs with Ghostferry and are confident that the dataset will be OK).

IMO given that this is an open source project, we should keep the flexibility of Ghostferry intact, if it is not too much work.